darasimiolaifa/epicmail
# Merge develop into gh-pages

This Pull Request merges develop into gh-pages:

The activities carried out includes:
- fixing the footer floating problem
- creating more pages to increase functionality and ease of use
- doing general UI clean up of code.

To test, take the following steps:
- git clone https://github.com/darasimiolaifa/epicmail
- cd epicmail
- npm run dev
- visit localhost:3000 in your browser
- Observe the functionality of pages
